dev/scripts/exglobal_atmos_analysis.sh

```diff
@@ -674,20 +670,21 @@ ${UNCOMPRESS} \$diag_file
 fnameges=\$(echo \$fname | sed 's/_ges//g')
 ${NMV} \$fname.\$fdate\$diag_suffix \$fnameges
 EOF
-    chmod 755 "${DATA}/unzip_radstat.sh"
+    chmod 755 "${DATA}/unzip_diag.sh"
 
     rm -f "${DATA}/cmdfile"
-    #shellcheck disable=SC2312
-    listdiag=$(tar -xvf radstat.gdas | cut -d' ' -f2 | grep _ges)
+    cpreq "${GRADSTAT}" radstat.tar
+    tar -xvf radstat.tar
+    listdiag=$(find ./ -path "./diag_*_ges.*" -type f)
```

Comment:
Suggest limiting the depth to 1 to limit impact to disk

```suggestion
    listdiag=$(find ./ -maxdepth 1 -path "./diag_*_ges.*" -type f)
```